package.json

```
@@ -1,22 +1,22 @@
 {
   "private": true,
   "name": "solc-bin",
+  "type": "module",
   "version": "0.0.0",
   "description": "Current and historical (emscripten) binaries for Solidity",
   "dependencies": {
     "semver": "^7.3.5"
   },
   "devDependencies": {
     "ethereumjs-util": "^7.1.3",
+    "blockstore-core": "^2.0.2",
     "ipfs-unixfs-importer": "^9.0.6",
```

Please also update `ipfs-unixfs-importer` to the latest version.
```suggestion
    "ipfs-unixfs-importer": "^11.0.1",
```